kaspa-wallet.js

```diff
@@ -500,19 +500,28 @@ class KaspaWalletCli {
 				}
 			})
 		program
-			.command('addresses <count> [search-address]')
+			.command('addresses')
+			.argument('<count>', 'number of addresses to show')
+			.argument('[search-address]', 'search address')
 			.option("--json", "output as json format")
-			.option("-s --start <start>", "output as json format")
-			.description('show wallet addresses', {
-				"search-address": "Search address",
-				"start":"Starting index"
-			})
+			.option("-s --start <start>", "starting index")
+			.option("--seed <seed_phrase>", "override wallet seed phrase")
```

Comment:
Solves the requirement.

Reply:
So the reason `--seed` command does not exist is that by entering the seed phrase on the command line, the seed ends up in the shell buffer, which in turn can be raised via a "history" type of command, making the seed insecure... 

This can be a serious problem for someone who does not understand security implications while holding large amounts of holdings with the seed...

I would accept this addition if there is some kind of warning attached to it - like a `.description()` that states that **_"this is not secure as this reveals your seed in the command line history buffer!"_**...

thoughts?

Reply:
After submitting this and having further discussions in the Kaspa Discord, I actually believe that this is not the correct way to move forward. Doubly so with the security concern.

Rather, I believe the correct way to proceed is to update https://wallet.kaspanet.io with a `Debug` button that enables searching for a specific `Receive Address`, leveraging the `Scan More Addresses` functionality.

Do you believe that is a viable option to solve [Tim's request](https://discordapp.com/channels/599153230659846165/755890250643144788/1042870419092414478)?

[Additional Context](https://discordapp.com/channels/599153230659846165/755890250643144788/1076077700705423430)